src/components/Data/Layout.vue

```diff
@@ -158,6 +159,10 @@ export default {
       this.dataNotFound = true
     },
     async fetchAllTheThings() {
+      if (!this.isAuthenticated) {
+        return
```

Comment:
🤔 I feel the day the AC will fall into this case outside a login or bootstrap sequence. The page will be empty, nothing will happen, no error in the console... Just crickets.
I suggest that, for the time being, we just add a warning in the console and when the moment comes, we handle this properly.

Reply:
I totally agree!

Reply:
Also, I found a way to test this case :)